integration/tests/user_scenario/basic_workflow/negative/neg_specify_deleted_forge_module.rb

```diff
@@ -0,0 +1,45 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-127 - C64288 - Attempt to Deploy Environment Specify Deleted Forge Module'
+
+#This test uses the regret module deleted from the acceptance forge (history at https://github.com/justinstoller/puppetlabs-regret/commits/master), with versions 0.1.0 - 0.4.0 deleted, effectively deleting the module.
+
+#Init
+git_environments_path = '/root/environments'
+last_commit = git_last_commit(master, git_environments_path)
+
+#Verification
+error_notification_regex = /Does 'puppetlabs-regret' have at least one published release?/
+
+#File
+puppet_file = <<-PUPPETFILE
+mod "puppetlabs/regret"
+PUPPETFILE
+
+puppet_file_path = File.join(git_environments_path, 'Puppetfile')
+
+#Teardown
+teardown do
+  clean_up_r10k(master, last_commit, git_environments_path)
+end
+
+#Setup
+step 'Stub Forge on Master'
+stub_forge_on(master)
+
+#Tests
+step 'Checkout "production" Branch'
+git_on(master, 'checkout production', git_environments_path)
+
+step 'Create "Puppetfile" for the "production" Environment'
+create_remote_file(master, puppet_file_path, puppet_file)
+
+step 'Push Changes'
+git_add_commit_push(master, 'production', 'Add module.', git_environments_path)
+
+#Tests
+step "Deploy production environment via r10k with specified module deleted"
+on(master, 'r10k deploy environment -p -v', :acceptable_exit_codes => [1]) do |result|
```

Comment:
The ":acceptable_exit_codes => [1]" argument can accept integers or arrays or integers. Since you're only passing a single integer into the array you can simply use a bare integer instead:

```ruby
:acceptable_exit_codes => 1
```